projects/observability/src/pages/apis/service-detail/instrumentation/service-instrumentation.component.ts

```diff
@@ -1,54 +1,40 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { map } from 'rxjs/operators';
 
 import { BreadcrumbsService } from '@hypertrace/components';
-import { BehaviorSubject, Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
 import { ServiceInstrumentationService } from './service-instrumentation.service';
-import { OrgScoreResponse, ServiceScoreResponse } from './service-instrumentation.types';
+import { ServiceScoreResponse } from './service-instrumentation.types';
 
 @Component({
   styleUrls: ['./service-instrumentation.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   providers: [ServiceInstrumentationService],
   template: `
     <main class="service-instrumentation" *ngIf="this.serviceScoreSubject | async">
-      <section class="overview">
-        <ht-service-instrumentation-total-score
-          [serviceScore]="(this.serviceScoreSubject | async)?.aggregatedWeightedScore"
-        ></ht-service-instrumentation-total-score>
-
-        <ht-service-instrumentation-org-score
-          [orgScore]="(this.orgScoreResponse$ | async)?.aggregatedWeightedScore"
-        ></ht-service-instrumentation-org-score>
-      </section>
-
-      <section class="checks-container">
-        <ht-service-instrumentation-category-card
-          *ngFor="let categoryScore of (serviceScoreSubject | async)?.qoiTypeScores"
-        ></ht-service-instrumentation-category-card>
-      </section>
+      <router-outlet></router-outlet>
     </main>
   `
 })
-export class ServiceInstrumentationComponent {
+export class ServiceInstrumentationComponent implements OnInit {
   public serviceScoreSubject: BehaviorSubject<ServiceScoreResponse | undefined> = new BehaviorSubject<
     ServiceScoreResponse | undefined
   >(undefined);
 
-  public orgScoreResponse$: Observable<OrgScoreResponse>;
-
   public constructor(
     private readonly breadcrumbsService: BreadcrumbsService,
     private readonly serviceInstrumentationService: ServiceInstrumentationService
   ) {
+    this.serviceScoreSubject = this.serviceInstrumentationService.serviceScoreSubject;
+  }
+
+  public ngOnInit(): void {
```

Comment:
Now that my PR is merged, can you try using the `getLastString` method to get the service name here?

Reply:
✅ Replaced `pipe` with `getLastBreadCrumbString`